rhel/SPECS/dbaas-tools.spec

```diff
@@ -0,0 +1,69 @@
+%define debug_package %{nil}
+%global provider_aws        github
+%global provider_tld_aws    com
+%global project_aws         kubernetes-sigs
+%global repo_aws            aws-iam-authenticator
+%global provider_prefix_aws %{provider_aws}.%{provider_tld_aws}/%{project_aws}/%{repo_aws}
```

Comment:
Do we really need it?

Reply:
I think we do, better not to hardcode, each component in this specfile will have such block

If it's too much for "not to hardcode" purpose, we can remove it.

Reply:
@EvgeniyPatlan I would propose to remove those variables, at least the once that we use only once like `provider_aws`

Reply:
LGTM

To be extra clear:
* When we add, for example, kubectl package, we'll add `provider_kubectl`/`provider_k8s`, right?

Reply:
@AlekSi I am also curious about this but last times I see that this becomes common practice :( So I would say let it be.